### DISCUSSION TITLE
Welcome to Plyr Discussions!

### DISCUSSION THREAD

**Discussion by sampotts** [sampotts/plyr#2090] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soundismore-coralie** [comment]
Hi,
I am not a developer, but I have to deal with some. Code is not completely unfamiliar to me. I am a sound designer, and work with audio programming.
I ran into plyr as I was looking for an sustainable and customizable solution for my Web site. Unfortunately, I don't have control over its development, and the webdesigner has been a bit cryptic about fureter developments. I am seeking basic nd objective information about the possibilities of plyr in terms of integration in a website.
If anyone would kindly reply I would greatly apreciate it.
